test-data/genops-try.test

```diff
@@ -30,11 +32,12 @@ L2: (handler for L1)
     r8 = getattr r6, r7
     r9 = py_call(r8, r5)
     r10 = cast(None, r9)
-L3:
-    clear_exc_info
+L3: (handler for L2)
+    restore_exc_info r4
```

Comment:
It's not obvious how exceptions raised in an except block (but that are not caught there) work. Where does the chaining of exceptions happen (there can be multiple tracebacks)? It would be good to document this.

Reply:
I'll add a comment, but the chaining of exceptions is done implicitly based on exc_info, it seems.